src/gui/base/NavButton.ts

```diff
@@ -30,12 +29,16 @@ export type NavButtonAttrs = {
 	small?: boolean
 	centred?: boolean
 	leftInjection?: () => Children
+	disableHoverBackground?: boolean
+	disabled?: boolean
 }
 
-const navButtonSelector = (vertical: boolean | undefined, centred?: boolean) =>
-	"a.nav-button.noselect.flex-no-shrink.items-center.click.plr-button.no-text-decoration.button-height.state-bg.border-radius" +
+const navButtonSelector = (vertical: boolean | undefined, noHoverBackground: boolean | undefined, disabled: boolean | undefined, centred?: boolean) =>
```

Comment:
these should be able to be declared as optional as well i think eg `noHoverBackground?: boolean`